pages/documentation/migration-V3.md

```diff
@@ -0,0 +1,886 @@
+     ```js
+     .pipe(
+      sass({
+        includePaths: [
+          PROJECT_SASS_SRC,
+          `${USWDS},
+          `${USWDS}/packages`,
+        ],
+      })
+    )
+    ```
+1. Recompile your Sass as usual. When it compiles, it is now using USWDS 3.0!
+</div>
+<!-- End USWDS Gulp section --> 
+
+<!-- Start USWDS Compile section -->
+<h4 class="usa-accordion__heading">
+  <button class="usa-accordion__button" aria-controls="m-a7">
+    If you're using USWDS Compile
+  </button>
+</h4>
+<div id="m-a7" class="usa-accordion__content site-prose" markdown="1">
+
+{:.site-terminal}
+```js
+/* gulpfile.js */
+
+const uswds = require("@uswds/compile");
+
+/**
+ * USWDS version
+ */
+
+uswds.settings.version = 3;
+```
+</div>
+<!-- End USWDS Compile section -->  
+
+<!-- Start custom Gulp workflow section -->
+<h4 class="usa-accordion__heading">
+  <button class="usa-accordion__button" aria-controls="m-a8">
+    If you're using a custom Gulp workflow
+  </button>
+</h4>
+<div id="m-a8" class="usa-accordion__content site-prose" markdown="1">
+TK   
+</div>
+<!-- End custom Gulp workflow section --> 
+
+<!-- Start custom Gulp workflow section -->
+<h4 class="usa-accordion__heading">
+  <button class="usa-accordion__button" aria-controls="m-a9">
+    If you're using Webpack
+  </button>
+</h4>
+<div id="m-a9" class="usa-accordion__content site-prose" markdown="1">
+TK   
+</div>
+<!-- End custom Gulp workflow section --> 
+
+</div> <!--End compiler accordion -->
+
+### 5. Update to Sass module syntax
+
+{:.border-top-2px.border-base-lighter.padding-top-1}
+USWDS 3.0 supports the deprecated Sass import syntax. You don't need to change to the new syntax in your project Sass to use USWDS 3.0. But once you update, we recommend updating to the new syntax as well.  You’ll need to both switch `@import` to `@use`, and then create `@use` project-specific settings.  In USWDS 3.0 with Sass modules, instead of having multiple theme files that contain every setting available to the design system, you only keep track of the settings you changed in a single file. 
+
+We recommend creating a new file to keep track of these changed settings. These instructions will help you create that file, format it properly, and use it in your project. 
+
+You can complete these changes by taking the following steps: 
+
+1. Replace all instances of `@import` with `@forward`, using find/replace in your Sass file.
+2. Locate your existing project theme files. These are the _uswds-theme files that you found in [Step 1: Check your current USWDS code and settings versions](#1-check-your-current-uswds-code-and-settings-versions). (If you already have your project-specific settings file in a single file, you can skip ahead Step 4. If not, you'll need to collect your project-specific settings in a single file.)
+3. In the project theme file directory directory, create a new file called `_uswds-theme.scss`.
+
+### 6. Find which settings from your theme files you've customized.
```

Comment:
@bonnieAcameron I am wondering if this step might actually be a better fit inside `5. Update to Sass module syntax` as item 4. It feels like an extension of updating to sass module syntax. Then '1. Once you have all your project-specific settings in a single file..' below would be step 5 of...step 5. It might be easier to call. Let me know if you have questions!

Reply:
Done.